### ISSUE TITLE
Investigate padding between help text and input

### ISSUE THREAD

**Issue by frosso** [Automattic/woocommerce-payments#1952]
## Description

For background info: p1622132280144800-slack-CU6SYV31A

> I wonder if there’s a way to increase the padding between the checkbox label and its description? 12 px would be perfect. I’ve also removed the link to Microsoft docs and updated the designs in Figma.

The padding comes from Gutenberg. Would it be fixed with a newer version of Gutenberg, without having to adjust all the `BaseControl` components?

**Comment by frosso** [comment]
This is the change that interests us: https://github.com/WordPress/gutenberg/pull/29550

With the Gutenberg release that includes that change (I just tested the last one), I can see the margin between the help text and the input being adjusted:
![Screen Shot 2021-06-07 at 3 35 53 PM](https://user-images.githubusercontent.com/273592/121084271-1245e080-c7a6-11eb-85ee-43a9c070e861.png)

**Comment by frosso** [comment]
Talked to the team about it.
Upgrading Gutenberg to the latest major version is definitely the way to go.

I created a ticket for the upgrade: https://github.com/Automattic/woocommerce-payments/issues/2085
Design asked to adjust the margin for the settings page, I opened a PR to tweak that: https://github.com/Automattic/woocommerce-payments/pull/2084

In the ticket I specified that the patch can be removed when the upgrade is done.
The patch should not conflict with the upgrade.
